src/utilities/extendSchema.js

```diff
@@ -257,14 +267,24 @@ export function extendSchema(
   // this scope and have access to the schema, cache, and newly defined types.
 
   function getMergedDirectives(): Array<GraphQLDirective> {
-    const existingDirectives = schema.getDirectives();
+    const existingDirectives = schema.getDirectives().map(extendDirectiveType);
```

Comment:
Maybe we shouldn't recreate standard directives like `@skip`, `@include` and `@deprecate`.
@leebyron What do you think?

Reply:
This is really elegant, so we can start with this. It would be a breaking change to disallow extending `@skip`/etc. but I like being able to extend them just like any other directive.